modules/cluster-logging-rn-5.3.10.adoc

```diff
@@ -0,0 +1,31 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-10"]
+= Logging 5.3.10
+This release includes link:https://access.redhat.com/errata/RHSA-2022:5908[RHSA-2022:5908-OpenShift Logging Bug Fix Release 5.3.10].
+
+[id="openshift-logging-5-3-10-bug-fixes"]
+== Bug fixes
+* https://bugzilla.redhat.com/show_bug.cgi?id=2100495[BZ-2100495]
+
```

Comment:
There is another JIRA for 5.3.10 [LOG-2817](https://issues.redhat.com/browse/LOG-2817) I don't see no-rn label on this one.

Reply:
Ignore this comment! This was a CVE :)

Reply:
> There is another JIRA for 5.3.10 [LOG-2817](https://issues.redhat.com/browse/LOG-2817) I don't see _no-rn_ label on this one.

Ignore this comment! This was a CVE :)

Reply:
[LOG-2817](https://issues.redhat.com/browse/LOG-2817) Is the private JIRA issue for https://bugzilla.redhat.com/show_bug.cgi?id=2100495[BZ-2100495]